### PR TITLE
BAH-3178 | ABHA Verification through mobile number

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: src/components/verifyHealthId/verifyHealthId.jsx [hunk 1]
```diff
@@ -187,35 +191,54 @@ const VerifyHealthId = () => {
...
+                        />}
+                    </div>
+                </div>}
+                {isVerifyThroughMobileNumberEnabled &&
+                <div>
+                    {!isMobileOtpVerified && <div className="alternative-text">
+                        OR
+                    </div>}
+                    <VerifyHealthIdThroughMobileNumber isDisabled={showAuthModes} setIsDisabled={setIsDisabled} setIsMobileOtpVerified={setIsMobileOtpVerified}
+                                                       ndhmDetails={ndhmDetails} setNdhmDetails={setNdhmDetails} setBack={setBack}/>
```
Comment (angshu) [inline src/components/verifyHealthId/verifyHealthId.jsx:240]: what is "setNdhmDetails" being set to? my impression is that it more of a callback which gets called when we fetch the profile from ABHA service.

File: src/components/Common/Util.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        value: patient.mobile
+    }] : undefined;
+    var address =  {
+        line: [patient?.address],
+        city: patient?.townName,
+        district: patient?.districtName,
+        state: patient?.stateName,
+        pincode: patient?.pincode
+    };
+    const ndhm = {
```
[the remark this answers is not in the document]
Comment (SanoferSameera) [reply src/components/Common/Util.jsx]: done

File: src/components/Common/Util.jsx
[the remark this answers is not in the document]
Comment (SanoferSameera) [reply src/components/Common/Util.jsx:1]: done

File: src/components/verifyHealthId/verifyHealthId.jsx (hunk 1 above)
Comment (SanoferSameera) [reply src/components/verifyHealthId/verifyHealthId.jsx:240]: Whenever we have abha profile, we will set it in that ndhmDetails

**Review by angshu — APPROVED** [review]

**Pull request merged**